Mathlib/Data/Finset/SymmDiff.lean

```diff
@@ -52,6 +52,18 @@ theorem image_symmDiff [DecidableEq β] {f : α → β} (s t : Finset α) (hf :
     (s ∆ t).image f = s.image f ∆ t.image f :=
   mod_cast Set.image_symmDiff hf s t
 
+/-- See `symmDiff_subset_sdiff'` for the swapped version of this. -/
+lemma symmDiff_subset_sdiff : s \ t ⊆ s ∆ t := subset_union_left
+
+/-- See `symmDiff_subset_sdiff` for the swapped version of this. -/
+lemma symmDiff_subset_sdiff' : t \ s ⊆ s ∆ t := subset_union_right
+
+lemma symmDiff_subset_union : s ∆ t ⊆ s ∪ t := symmDiff_le_sup (α := Finset α)
+
+lemma symmDiff_eq_union_iff (s t : Finset α) : s ∆ t = s ∪ t ↔ Disjoint s t := symmDiff_eq_sup s t
```

Comment:
For reviewers: this is given explicit arguments for consistency with `symmDiff_eq_sup`, which also has explicit arguments.